### PR TITLE
Handle CONCAT/LENGTH functions on supported nullable types

### PR DESCRIPTION
**Pull request by mkatri**

### CODE REVIEW

File: cel2sql_test.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ func TestConvert(t *testing.T) {
 			decls.NewVar("height", decls.Double),
 			decls.NewVar("string_list", decls.NewListType(decls.String)),
 			decls.NewVar("string_int_map", decls.NewMapType(decls.String, decls.Int)),
+			decls.NewVar("nullable_string", decls.NewWrapperType(decls.String)),
```
Comment (starius) [inline cel2sql_test.go:36]: Please add a test for nullable bytes as well.

File: cel2sql.go [hunk 2]
```diff
@@ -922,6 +920,14 @@ func IsListType(typ *exprpb.Type) bool {
 	return ok
 }
 
+func IsStringType(typ *exprpb.Type) bool {
```
Comment (starius) [inline cel2sql.go:985]: Google SQL also supports nullable arrays:
https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#array_nulls
Should we support them as well? I guess it requires similar changes in `IsListType` function.

**Review by starius — APPROVED** [review]
LGTM, few notes

File: cel2sql_test.go (hunk 1 above)
Comment (mkatri) [reply cel2sql_test.go:36]: done

File: cel2sql.go (hunk 2 above)
Comment (mkatri) [reply cel2sql.go:985]: google sql does but I can't find a way to represent a nullable list in cel
wrapper only works for primitive types (doesn't include lists) 
I tried google.protobuf.ListValue but CEL mistakenly converts that to a non-nullable checked list type (https://github.com/google/cel-go/blob/a465d9346efd194352045078c08649a16a39d23d/common/types/pb/checked.go#L46)

**Pull request merged**